src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrRadioPageTask.java

```diff
@@ -0,0 +1,43 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import static de.mediathekview.mserver.base.Consts.ATTRIBUTE_HREF;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.WdrConstants;
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDTO;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.commons.lang3.text.WordUtils;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class WdrRadioPageTask extends AbstractDocumentTask<WdrTopicUrlDTO, CrawlerUrlDTO> {
+
+  private static final String SELECTOR_TOPIC = "h2 > a";
+  
+  public WdrRadioPageTask(AbstractCrawler aCrawler, ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)